gulpfile.js

```diff
@@ -47,24 +35,20 @@ var argv = yargs
         .option('clear_datastore', {
           describe: 'Whether to clear all data storage'
         })
-        .option('use_minification', {
-          describe: 'Whether to build with minification'
-        })
         .option('prod_env', {
           describe: 'Whether to run server in prod mode'
         })
         .demand(['gae_devserver_path'])
         .argv;
     }).argv;
-var concat = require('gulp-concat');
+
 var gulp = require('gulp');
+var concat = require('gulp-concat');
+var shell = require('gulp-shell');
```

Comment:
On searching for gulp-shell one of the first results tells me that "gulp-shell has been blacklisted": https://stackoverflow.com/questions/29511491/running-a-shell-command-from-gulp

Any thoughts on this? Worth using the alternative mentioned there instead?

Reply:
Oh, I didn't know there even are some blacklisted plugins. I reworked it.